packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -408,6 +416,9 @@ class SliderThemeData extends Diagnosticable {
       thumbShape: t < 0.5 ? a.thumbShape : b.thumbShape,
       valueIndicatorShape: t < 0.5 ? a.valueIndicatorShape : b.valueIndicatorShape,
       showValueIndicator: t < 0.5 ? a.showValueIndicator : b.showValueIndicator,
+      valueIndicatorTextStyle: (a.valueIndicatorTextStyle != null && b.valueIndicatorTextStyle != null)
+          ? TextStyle.lerp(a.valueIndicatorTextStyle, b.valueIndicatorTextStyle, t)
```

Comment:
this doesn't seem right...

Reply:
Well, but I think it's the best I can do, because I can't get the default text style from the theme, because I don't have a context (and lerp doesn't like lerping between nulls).

Reply:
I think you can just require that the text style is non-null by defaulting it to the empty text style. Combined with merging the text style, you get the effect you want.

Reply:
So you're suggesting that the text style should really just be the "text style that is merged into the default value" rather than specifying the entire text style?  I think I'd rather require it and have it specify the whole enchilada, and have them find the way to merge/copy from the default, because otherwise they would be surprised by the default (the empty text style).